tests/msbr_processes.json

```diff
@@ -0,0 +1,103 @@
+{
+  "fuel": {
+    "extraction_processes": {
+      "core_inlet": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 0
+      },
+      "core_outlet": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 0
+      },
+      "heat_exchanger": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 9920000,
+        "volume": 13700000
+      },
+      "bypass": {
+        "capacity": 9920000,
+        "efficiency": {},
+        "mass_flowrate": 8928000,
+        "volume": 1000000000
+      },
+      "sparger": {
+        "capacity": 9920000,
+        "efficiency": {
+          "Kr": 0,
+          "Xe": 0,
+          "H": 0
+        },
+        "mass_flowrate": 9920000,
+        "volume": 10000000
+      },
+      "entrainment_separator": {
+        "capacity": 9920000,
+        "efficiency": {
+          "Kr": 0.9115218695901359,
+          "Xe": 0.9148573428740562
+        },
```

Comment:
Or is the gas separator where you account for removal, not the sparger?  Does this separator not also remove the helium gas from the liquid salt before it hits a liquid pump?

Reply:
As you can see, the entrainment separator does not remove helium. I think the way the reprocessing system files are set up is the least understood/documented aspect of SaltProc right now. I'm going to be adding some guides on this.